gap-analysis/index.md

```diff
@@ -369,51 +369,51 @@ Such an attribute could be used by an authoring tool, for example, to encode the
 
 If (0,5) had another meaning, such as an open interval, or an ordered pair, or greatest common divisor, a value other than “point” would be used in the intent attribute.
 
+Here is some potential markup using `intent` for the remaining two examples.
+<details markdown="1">
+<summary><span markdown="1">Line segment example $\overline{A'B'}$ </span></summary>
 
-The line segment from the point A' to the point B' could be encoded as:
-
+This example shows the need to use nested `intent`s.
 ```xml
-<mover intent="segment($x,$y)">
+<mover intent="line-segment($x,$y)">
   <mrow>
-    <msup arg="x" intent="prime($1)">
+    <msup arg="x" intent="primed($1)">
       <mi arg="1">A</mi>
       <mo>&#x2032;</mo>
     </msup>
     <mo>&#x2063;</mo>
-    <msup arg="y" intent="prime($1)">
+    <msup arg="y" intent="primed($1)">
       <mi arg="1">B</mi>
       <mo>&#x2032;</mo>
     </msup>
   </mrow>
   <mo>¯</mo>
 </mover>
 ```
-
-where the name “segment” is used here to mean line segment, and the name “prime” is used to denote a primed variable symbol.
-
-Other meanings for the overbar notation would replace “segment” with another operator in the intent attribute.
+</details>
 
 
-The x-coordinate of the point B' could be encoded as:
+<details markdown="1">
+<summary><span markdown="1">X-coordinate example $B'_x$ </span></summary>
+This is similar to the line segment example in terms of complexity. Note that the second argument is meant to be spoken first.
 
 ```xml
-<msub intent="index($p,$v)">
-  <msup arg="p" intent="prime($1)">
+<msub intent="coordinate-of($p,$v)">
```

Comment:
the `-of` doesn't follow what we've discussed before, and it will make it harder to reuse the concept name as an anchor later. `coordinate($1,$2)` would need a speech hint to be read as `$1-coordinate of $2`, if that's the goal here.

The example is a little tortured, since in a real-world formula people are likely to just say "B x prime" or "B prime x" and move on with the rest of the expression. Feels closer to the `aria-describedby` descriptions than to needed speech... So maybe the ultimate comment is that we should leave this as raw presentation and focus elsewhere.